tools/stress-cluster/cluster/kubernetes/stress-test-addons/deploy.ps1

```diff
@@ -0,0 +1,43 @@
+function Run()
+{
+    Write-Host "`n==> $args`n" -ForegroundColor Green
+    $command, $arguments = $args
+    & $command $arguments
+    if ($LASTEXITCODE) {
+        Write-Error "Command '$args' failed with code: $LASTEXITCODE" -ErrorAction 'Continue'
+    }
+}
+function RunOrExitOnFailure()
+{
+    Run @args
+    if ($LASTEXITCODE) {
+        exit $LASTEXITCODE
+    }
+}
+$ErrorActionPreference = 'Stop'
+Set-PSDebug -trace 1
+$subscriptionId = "2cd617ea-1866-46b1-90e3-fffb087ebf9b"
+$env:AZURE_STORAGE_ACCOUNT="stresstestcharts"
+$env:AZURE_STORAGE_KEY=$(RunOrExitOnFailure az storage account keys list --subscription $subscriptionId --account-name $env:AZURE_STORAGE_ACCOUNT -o tsv --query '[0].value')
+
+Remove-Item -Force ./*.tgz
+
+RunOrExitOnFailure helm package  .
+RunOrExitOnFailure helm repo index --url https://stresstestcharts.blob.core.windows.net/helm/ --merge index.yaml .
+
+# The index.yaml in git should be synced with the index.yaml already in blob storage
+# az storage blob download -c helm -n index.yaml -f index.yaml
+
+$files = (Get-Item *.tgz).Name
+$confirmation = Read-Host "Are you sure you want to upload files: $files`? [y/n]"
```

Comment:
"Are you sure" implies something destructive (e.g. are you sure you want to delete), so I'd phrase it differently and also be more specific about what action is about to be taken.

```suggestion
$confirmation = Read-Host "Do you want to update the helm repository to add $files`? [y/n]"
```